src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -807,7 +947,10 @@ def forward(self, hidden_states: List[torch.Tensor]) -> List[torch.Tensor]:
             raise ValueError("The number of hidden states should be equal to the number of neck hidden sizes.")
 
         # postprocess hidden states
-        features = self.reassemble_stage(hidden_states)
+        if self.is_using_hybrid:
+            features = self.reassemble_stage(hidden_states, ignore_index=[0, 1])
```

Comment:
```suggestion
            features = self.reassemble_stage(hidden_states, ignore_index=config.hybrid_ignore_index])
```
this is hardcoded and can be changed